presidio-analyzer/tests/test_credit_card_recognizer.py

```diff
@@ -0,0 +1,196 @@
+# https://www.datatrans.ch/showcase/test-cc-numbers
+# https://www.freeformatter.com/credit-card-number-generator-validator.html
+from unittest import TestCase
+
+from analyzer.predefined_recognizers import CreditCardRecognizer
+
+entities = ["CREDIT_CARD"]
+credit_card_recognizer = CreditCardRecognizer()
+
+
+class TestCreditCardRecognizer(TestCase):
+
+    def test_valid_credit_cards(self):
+        # init
+        number1 = '4012888888881881'
+        number2 = '4012-8888-8888-1881'
+        number3 = '4012 8888 8888 1881'
+
+        results = credit_card_recognizer.analyze('{} {} {}'.format(number1, number2, number3), entities)
+
+        assert len(results) == 3
```

Comment:
I would recommend adding a function:
validate_recognizer_result(result, expected_entity_type, expected_start, expected_end, expected_score)
and validate_recognizer_results(results, expected_results) - to also compare an array of results
They can be implemented in a recognizer_result_validation.py under the tests folder

No need to do it in this PR, but I recommend adding a Task for it